app/views/site/contribute.erb

```diff
@@ -0,0 +1,168 @@
+        <li><strong>Language track repositories</strong>: Each language has its own repository with available problems.</li>
+        <li><strong>Problem metadata</strong>: Contain the language-specific implementations of exercises. Each language has its own repository.</li>
+        <li><strong>Problems API</strong>: Delivers exercises, consumed by the Exercism command-line client, as well as the Exercism website.</li>
+        <li><strong>Rikki- the robot</strong>: Provides helpful, friendly, automated feedback on certain exercises.</li>
+      </ol>
+    </p>
+  </section>
+
+  <hr>
+
+  <section class="contributing-areas">
+    <div class="row">
+      <div class="col-md-4">
+        <div class="contributing-section">
+          <h2>Make Up Exercises</h2>
+
+          <p>
+            We're always looking to add new exercises. Have an idea? Suggest it in the <a href="https://github.com/exercism/x-common">metadata repository</a>.
+          </p>
+
+           <p>
+            Check out the section about <a href="https://github.com/exercism/x-common/blob/master/CONTRIBUTING.md#implementing-a-completely-new-exercise">implementing a new exercise</a> in the Contributing Guide.
+          </p>
+        </div>
+      </div>
+
+      <div class="col-md-4">
+        <div class="contributing-section">
+          <h2>Translate Exercises</h2>
+
+          <p>
+            Another great way to add new exercises is port an existing exercise to one of the other languages. This is easier than making up an exercise from scratch, since other people have thought through which test cases are needed.
+          </p>
+
+          <p>
+            Track maintainers are more than happy to give you all the support you need to successfully contribute.
+          </p>
+
+          <p>
+            Check out the detailed guide <a href="https://github.com/exercism/x-common/blob/master/CONTRIBUTING.md#porting-an-exercise-to-another-language-track">here</a> if you want to give it a try!
+          </p>
+        </div>
+      </div>
+
+      <div class="col-md-4">
+        <div class="contributing-section">
+          <h2>Review Pull Requests</h2>
+
+          <p>
+            We want the quality of the exercises to be as high as possible (and reviewing pull requests is a great way to both learn and teach a programming language).
+          </p>
+
+          <p>
+            We look at all sorts of things: style, idiomatic use of language features, naming, missing test cases, and redundant test cases. We also try to avoid forcing the solution to take a particular approach. Subscribe to the language track repositories of your favorite languages so you get notifications about new issues and pull requests. The track maintainers will greatly appreciate your help!
+          </p>
+
+          <p>
+             You can search for language track repositories <a href="https://github.com/exercism/">here</a>.
```

Comment:
I think this would be better than just pointing to the org on github, because there's a lot of messy legacy stuff there, as well as tools that they shouldn't be confronted with, and the sort order isn't configurable.